java-client/src/main/java/com/cumulocity/sdk/client/messaging/notifications/TokenClaims.java

```diff
@@ -45,4 +49,14 @@ public long getIat() {
     public long getExp() {
         return this.exp;
     }
+
+    @JSONProperty(value = "shared", ignoreIfNull = true)
+    public boolean getShared() {
```

Comment:
Should be isShared() and isNonPeristent()?

Reply:
I'll try it (check it works once changed). This hasn't changed from original PR but overlooked I guess (it was your PR so maybe forgotten/overlooked as not explicitly reviewed). Should affect the core PR build but will need change ready so we can merge this promptly after core.

Reply:
yes, according to java beans "small" boolean should be prefixed with `is`, otherwise we my have issues with serialization/deserialization